torch/_dynamo/variables/dicts.py

```diff
@@ -398,8 +389,6 @@ def _add(self, item):
                         if alias_guard:
                             install_guard(e.vt.source.make_guard(alias_guard))
 
-        return self.items
```

Comment:
No call to mutation() `_add` function?

Reply:
So this is also called in the constructor, so I added a mutation() call in pop and add method handling.